tests/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# This is the cmake build file for the tests directory of the
+# NCEPLIBS-g2c_UTILS project.
+#
+# Ed Hartnett, 7/11/21
+
+if(CMAKE_Fortran_COMPILER_ID MATCHES "^(Intel)$")
+  set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -r8 -convert big_endian -assume byterecl")
+elseif(CMAKE_Fortran_COMPILER_ID MATCHES "^(GNU)$")
+  set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -ffree-line-length-0 -fdefault-real-8 -fconvert=big-endian")
+endif()
+
+include_directories(${PROJECT_SOURCE_DIR})
```

Comment:
Please use `target_inlcude_directories()` command to add include directories to individual targets.

Reply:
Even in cases where all targets need this include directory?

Reply:
yes.

Reply:
I guess I was hoping for a little more explanation as to why. Seems like this would just add a line to CMakeLists.txt for every test, when all tests can be handled with just one line...

Reply:
https://cmake.org/cmake/help/latest/command/include_directories.html

Reply:
OK, will do.